{
  "url": "https://api.github.com/repos/flashbots/mev-boost/issues/314",
  "repository_url": "https://api.github.com/repos/flashbots/mev-boost",
  "labels_url": "https://api.github.com/repos/flashbots/mev-boost/issues/314/labels{/name}",
  "comments_url": "https://api.github.com/repos/flashbots/mev-boost/issues/314/comments",
  "events_url": "https://api.github.com/repos/flashbots/mev-boost/issues/314/events",
  "html_url": "https://github.com/flashbots/mev-boost/issues/314",
  "id": 1371983576,
  "node_id": "I_kwDOGZSKs85Rxs7Y",
  "number": 314,
  "title": "Port forwarding broken in `docker run` docs @ https://hub.docker.com/r/flashbots/mev-boost",
  "user": {
    "login": "analogue",
    "id": 26757,
    "node_id": "MDQ6VXNlcjI2NzU3",
    "avatar_url": "https://avatars.githubusercontent.com/u/26757?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/analogue",
    "html_url": "https://github.com/analogue",
    "followers_url": "https://api.github.com/users/analogue/followers",
    "following_url": "https://api.github.com/users/analogue/following{/other_user}",
    "gists_url": "https://api.github.com/users/analogue/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/analogue/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/analogue/subscriptions",
    "organizations_url": "https://api.github.com/users/analogue/orgs",
    "repos_url": "https://api.github.com/users/analogue/repos",
    "events_url": "https://api.github.com/users/analogue/events{/privacy}",
    "received_events_url": "https://api.github.com/users/analogue/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "open",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 0,
  "created_at": "2022-09-13T20:40:31Z",
  "updated_at": "2022-09-13T20:41:06Z",
  "closed_at": null,
  "author_association": "NONE",
  "active_lock_reason": null,
  "body": "Since `mev-boost` defaults to listening on `localhost:18850`, the guidance provided @ https://hub.docker.com/r/flashbots/mev-boost fails in a surprising way since `mev-boost` needs to listen on `0.0.0.0:18550` for the `docker run ... -p 18550:18550` port forward to work correctly.\r\n\r\nI'm guessing changing the default for `mev-boost` from `localhost:18550` to `0.0.0.0:18550` is probably not going to go over too well so the alternative is to simply update the documentation @ https://hub.docker.com/r/flashbots/mev-boost across all nets (mainnet, goerli, etc).\r\n\r\nBroken docs:\r\n```\r\ndocker run -p 18550:18550 flashbots/mev-boost:latest -mainnet -relay-check -relays https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net\r\n```\r\n\r\nFixed docs with `-addr` flag:\r\n```\r\ndocker run -p 18550:18550 flashbots/mev-boost:latest -mainnet -addr 0.0.0.0:18550 -relay-check -relays https://0xac6e77dfe25ecd6110b8e780608cce0dab71fdd5ebea22a16c0205200f2f8e2e3ad3b71d3499c54ad14d6c21b41a37ae@boost-relay.flashbots.net\r\n```\r\n\r\nI would normally submit a PR, but I don't see the documentation in question under source control.",
  "closed_by": null,
  "reactions": {
    "url": "https://api.github.com/repos/flashbots/mev-boost/issues/314/reactions",
    "total_count": 0,
    "+1": 0,
    "-1": 0,
    "laugh": 0,
    "hooray": 0,
    "confused": 0,
    "heart": 0,
    "rocket": 0,
    "eyes": 0
  },
  "timeline_url": "https://api.github.com/repos/flashbots/mev-boost/issues/314/timeline",
  "performed_via_github_app": null,
  "state_reason": null
}
[

]
